### PR TITLE
RSKIP188 - Precompiled Contracts for BLS12-381 Curve Operations

### PR DESCRIPTION
**Pull request by fedejinich**
This PR adds a set of precompiled contracts for BLS12 curve operations.

### CODE REVIEW

File: IPs/RSKIP188.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Title
```
Comment (SergioDemianLerner) [inline IPs/RSKIP188.md:1]: Title "Precompile for BLS12-381 curve operations" must be in this first line also.

File: IPs/RSKIP188.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+|RSKIP          | 188 |
+| :------------ |:-------------|
+|**Title**      |Precompile for BLS12-381 curve operations |
+|**Created**    |20-11-2020 |
+|**Author**     |FJ |
+|**Purpose**    |USa |
```
Comment (SergioDemianLerner) [inline IPs/RSKIP188.md:8]: USa -> Usa

File: IPs/RSKIP188.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+## Motivation
+
+Motivation of this precompile:
+- Mantain compatibility between RSK and Ethereum
+- Add a cryptographic primitive that allows to get 120+ bits of security for operations over pairing friendly curve compared to the existing BN254 precompile that only provides 80 bits of security.
+
+## Specification
+
+If `blockNumber >= iris` we introduce nine separate precompiles to perform the following operations:
```
Comment (SergioDemianLerner) [inline IPs/RSKIP188.md:39]: Here is should not mention Iris, since this RSKIP was removed from Iris network upgrade.

Comment (SergioDemianLerner) [reply IPs/RSKIP188.md:39]: It could mention "N" as the activation height

File: IPs/RSKIP188.md (hunk 1 above)
Comment (fedejinich) [reply IPs/RSKIP188.md]: done :D

File: IPs/RSKIP188.md (hunk 3 above)
Comment (fedejinich) [reply IPs/RSKIP188.md]: done :)

File: IPs/RSKIP188.md (hunk 2 above)
Comment (fedejinich) [reply IPs/RSKIP188.md]: done :P

**Review by smishraIOV — APPROVED** [review]

**Pull request merged**